examples/SquareLine/WiFiClock/WiFiClock.ino

```diff
@@ -0,0 +1,980 @@
+    _ui_flag_modify(ui_ImageSnow, LV_OBJ_FLAG_HIDDEN, _UI_MODIFY_FLAG_ADD);
+    _ui_flag_modify(ui_ImageSunny, LV_OBJ_FLAG_HIDDEN, _UI_MODIFY_FLAG_ADD);
+    _ui_flag_modify(ui_ImageThunderstor, LV_OBJ_FLAG_HIDDEN, _UI_MODIFY_FLAG_ADD);
+    _ui_flag_modify(ui_ImageAtmosphere, LV_OBJ_FLAG_HIDDEN, _UI_MODIFY_FLAG_ADD);
+    _ui_flag_modify(ui_ImageDrizzle, LV_OBJ_FLAG_HIDDEN, _UI_MODIFY_FLAG_ADD);
+    if(Weather == "Clear") {
+        _ui_flag_modify(ui_ImageSunny, LV_OBJ_FLAG_HIDDEN, _UI_MODIFY_FLAG_REMOVE);
+    } else if(Weather == "Clouds") {
+        _ui_flag_modify(ui_ImageCloudy, LV_OBJ_FLAG_HIDDEN, _UI_MODIFY_FLAG_REMOVE);
+    } else if(Weather == "Rain") {
+        _ui_flag_modify(ui_ImageRain, LV_OBJ_FLAG_HIDDEN, _UI_MODIFY_FLAG_REMOVE);
+    } else if(Weather == "Snow") {
+        _ui_flag_modify(ui_ImageSnow, LV_OBJ_FLAG_HIDDEN, _UI_MODIFY_FLAG_REMOVE);
+    } else if(Weather == "Drizzle") {
+        _ui_flag_modify(ui_ImageDrizzle, LV_OBJ_FLAG_HIDDEN, _UI_MODIFY_FLAG_REMOVE);
+    } else if(Weather == "Thunderstorm") {
+        _ui_flag_modify(ui_ImageThunderstor, LV_OBJ_FLAG_HIDDEN, _UI_MODIFY_FLAG_REMOVE);
+    } else if(Weather == "Mist" || Weather == "Smoke" || Weather == "Haze" || Weather == "Dust" || \
+                Weather == "Fog" || Weather == "Sand" || Weather == "Dust") {
+        _ui_flag_modify(ui_ImageAtmosphere, LV_OBJ_FLAG_HIDDEN, _UI_MODIFY_FLAG_REMOVE);
+    }
+
+    lvgl_port_unlock();
+}
+
+void onWeatherUpdate(lv_timer_t *timer)
+{
+    if(wifi_connected_flag) {
+        lv_obj_t *ui_Labelweather = (lv_obj_t *) timer->user_data;
+        lv_label_set_text_fmt(ui_Labelweather, "%s", Weather);
+        if(prev_weather != Weather) {
+            WeatherMod(Weather);
+            prev_weather = Weather;
+        }
+    }
+}
+
+void onTemperatureUpdate(lv_timer_t *timer)
+{
+    if(wifi_connected_flag) {
+        lv_obj_t *ui_Labeltemperature = (lv_obj_t *) timer->user_data;
+        if(temperature != 999) {
+            sprintf(temperature_buffer, "%d℃", temperature);
+        }
+        lv_label_set_text_fmt(ui_Labeltemperature, "%s", temperature_buffer);
+    }
+}
+
+void initializeUITimer()
+{
+    lv_timer_t *LabelTime_Wifitimer = lv_timer_create(onTimerClockUpdate, 1000, (void *) ui_LabelTime);
+    onTimerClockUpdate(LabelTime_Wifitimer);
+    lv_timer_t *Labelcalendar_timer = lv_timer_create(onCalendarUpdate, 1000, (void *) ui_Labelcalendar);
+    onCalendarUpdate(Labelcalendar_timer);
+    lv_timer_t *Labelweek_timer = lv_timer_create(onWeekUpdate, 1000, (void *) ui_Labelweek);
+    onWeekUpdate(Labelweek_timer);
+    lv_timer_t *Labelweather_timer = lv_timer_create(onWeatherUpdate, 1000, (void *) ui_Labelweather);
+    onWeatherUpdate(Labelweather_timer);
+    lv_timer_t *Labeltemperature_timer = lv_timer_create(onTemperatureUpdate, 1000, (void *) ui_Labeltemperature);
+    onTemperatureUpdate(Labeltemperature_timer);
+}
+
+void handleWifiListItemClick(lv_event_t * e)
```

Comment:
Add comments for these codes like the above.  BTW, please add a note to explain where we need to use these functions (Though I know that in `ui`), since the user can't find any calling for these functions in the `.ino` file.